apps/site/src/app/schedule/schedule.tsx

```diff
@@ -0,0 +1,23 @@
+import { PortableText } from "@portabletext/react";
+import ClipboardSchedule from "./ClipboardSchedule";
+import { getSchedule } from "./getSchedule";
+
+import styles from "./page.module.scss";
```

Comment:
Chore: could we also rename the stylesheet to match the component, i.e. `Schedule.module.scss`